index.html

```diff
@@ -440,6 +443,13 @@ <h2>Hang Tight</h2>
 		model.update();
 		view.cellUpdate(world); 
 
+		if (autosave_period) {
```

Comment:
we'll probably need more atomic pull requests in the future, but I'm ok with this

Reply:
Understood. I will not combine multiple functional changes into one pull-req in the future.